cmd/ocm-backplane/login/login_test.go

```diff
@@ -387,4 +394,40 @@ var _ = Describe("Login command", func() {
 		})
 
 	})
+
+	Context("check GetRestConfigAsUser", func() {
+
+		It("should fail if generated config do not contains provided username without elevationReasons",func () {
```

Comment:
The test name seems a bit confusing, I would reverse it: it should succeed if ... as you are also testing 
 `Expect(err).To(BeNil())` below. A test that mentions "should fail if..." should check for a failure.